permissions/plugin-jobConfigHistory.yml

```
@@ -1,6 +1,6 @@
 ---
-name: "jobConfigHistory"
-github: "jenkinsci/jobConfigHistory-plugin"
+name: "job-config-history"
```

name == artifactId
https://github.com/jenkins-infra/repository-permissions-updater#managing-permissions

```suggestion
name: "jobConfigHistory"
```

---

🤦🏻 me